package.json

```diff
@@ -5,8 +5,9 @@
   "main": "lib/index.js",
   "types": "lib/src/index.d.ts",
   "scripts": {
-    "build": "webpack --env dev && webpack --env build && npm run test",
+    "build": "webpack --env dev && webpack --env build",
```

Comment:
I don't quite understand what is wrong with running tests after build. But I have no problem not doing it either

Reply:
no no, there is no issue. there is a need to have run build before test on `npm test`, thats why i take it out and place it under pretest command, if you think we need to run test after build, i can restructure it.

Reply:
Nope, all good; approved PR